ast/functions.go

```diff
@@ -329,12 +329,18 @@ func (n *FuncCallExpr) Format(w io.Writer) {
 // specialFormatArgs formats argument list for some special functions.
 func (n *FuncCallExpr) specialFormatArgs(w io.Writer) bool {
 	switch n.FnName.L {
-	case "date_add":
+	case DateAdd, DateSub, AddDate, SubDate:
 		n.Args[0].Format(w)
 		fmt.Fprintf(w, ", INTERVAL ")
 		n.Args[1].Format(w)
 		fmt.Fprintf(w, " %s", n.Args[2].GetDatum().GetString())
 		return true
+	case TimestampAdd, TimestampDiff:
```

Comment:
ditto

Reply:
I have add cases for `date_add` and `timestampdiff`. `date_sub` is same with `date_add`, and `timestampadd` is same with `timestampdiff`, so I think we can ignore them.